src/HealthKit/HKObjectType.cs

```diff
@@ -9,41 +9,46 @@
 //
 // Copyright 2014-2015 Xamarin
 //
+
+#nullable enable
+
 using System;
 using Foundation;
 namespace HealthKit
 {
+#if NET || WATCH
```

Comment:
Why this?

Reply:
Sorry, I made this change due to these selectors being Internal for NET or WATCH and being obsoleted for everything else: https://github.com/xamarin/xamarin-macios/blob/main/src/healthkit.cs#L1098-L1135

Looks like the ApiDiff shows this as a breaking change though so maybe this wasn't the right thing to do? @mandel-macaque

Reply:
It was not, has to be reverted too.